backend/rates/history_test.go

```diff
@@ -87,8 +87,8 @@ func TestUpdateHistory(t *testing.T) {
 		},
 	}
 
-	start, end := time.Unix(wantStartUnix, 0), time.Unix(wantEndUnix, 0)
-	n, err := updater.updateHistory(context.Background(), "btc", "USD", start, end)
+	g := fixedTimeRange(time.Unix(wantStartUnix, 0), time.Unix(wantEndUnix, 0))
```

Comment:
g?? :joy:

Reply:
well, it's "time ran**g**e" and `t` is already taken by the testing.T :)
but really, it's in a test and used only on the very next line. doesn't deserve a longer name imho.

Reply:
lol.

still i have very much disagree. also in tests you write it once and read it many times, a long name takes no effort compared to the reward.

alternative is also to just inline the whole thing and make the function call a multi-liner.

Reply:
both a long name used only on the very next line and a multi line expression are harder to read imho.

Reply:
for me it's the opposite, i always do a double take on a single letter varname, panicking what it means, and having a hard time searching for it to see where it is used when I feel the need to search for it.